packages/react-mutation-mapper/src/component/oncokb/OncoKbCardBody.tsx

```diff
@@ -76,13 +79,21 @@ const TabTitle: React.FunctionComponent<{
 }> = props => {
     const title = DATA_TYPE_TO_TITLE[props.type];
     const icon = props.displayHighestLevelInTabTitle ? (
-        <i
-            className={annotationIconClassNames(
-                props.type,
-                calcHighestIndicatorLevel(props.type, props.indicator),
-                props.indicator
-            )}
-        />
+        props.type === OncoKbCardDataType.BIOLOGICAL ? (
+            <OncogenicIcon
+                oncogenicity={props.indicator?.oncogenic || ''}
+                showDescription
```

Comment:
I know it is minor but a prop without a value always bothers me
```suggestion
                showDescription={true}
```

Reply:
Is this the convention in portal? We don't specify true in oncokb anymore

Reply:
I don't think we have a strict rule about this, but I think the majority of the boolean props in portal codebase have the explicit `={true}`.